<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ejemploPp</class>
 <widget class="QMainWindow" name="ejemploPp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ALSE Ejmeplo </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Matriz A</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLineEdit" name="txt_A11">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="txt_A12">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLineEdit" name="txt_A21">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="txt_A22">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbo_Operacion">
          <item>
           <property name="text">
            <string>+</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>-</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>*</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>/</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Matriz B</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLineEdit" name="txt_B11">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="txt_B12">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLineEdit" name="txt_B21">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="txt_B22">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Matriz Resultado</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLineEdit" name="txt_R11">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="txt_R12">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLineEdit" name="txt_R21">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="txt_R22">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="cmd_Operar">
          <property name="text">
           <string>Operar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
   </widget>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEditar"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionAbrir</sender>
   <signal>triggered()</signal>
   <receiver>cmd_Operar</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
